tests/phpunit/tests/rest-api/rest-comments-controller.php

```diff
@@ -4069,4 +4069,88 @@ public function data_note_status_provider() {
 			'reopen'   => array( 'reopen' ),
 		);
 	}
+
+	/**
+	 * Test children link for note comment type. Based on test_get_comment_with_children_link.
+	 *
+	 * @ticket 64152
+	 */
+	public function test_get_note_with_children_link() {
+		$comment_id_1 = self::factory()->comment->create(
+			array(
+				'comment_approved' => 0,
+				'comment_post_ID'  => self::$post_id,
+				'user_id'          => self::$subscriber_id,
+				'comment_type'     => 'note',
+			)
+		);
+
+		self::factory()->comment->create(
+			array(
+				'comment_approved' => 0,
+				'comment_parent'   => $comment_id_1,
+				'comment_post_ID'  => self::$post_id,
+				'user_id'          => self::$subscriber_id,
+				'comment_type'     => 'note',
+			)
+		);
+		wp_set_current_user( self::$admin_id );
+		$request = new WP_REST_Request( 'GET', sprintf( '/wp/v2/comments/%s', $comment_id_1 ) );
+		$request->set_param( 'type', 'note' );
+		$request->set_param( 'context', 'edit' );
+		$response = rest_get_server()->dispatch( $request );
+		$this->assertSame( 200, $response->get_status() );
+
+		$this->assertArrayHasKey( 'children', $response->get_links() );
+	}
+
+	/**
+	 * Test that children note links are properly embedded in the REST API response.
+	 *
+	 * @ticket 64145
+	 */
+	public function test_get_note_with_embedded_children() {
```

Comment:
I think this test is the same as the one `above.` We're asserting that links contain `children` link and it has the proper shape.

We could combine them.

Reply:
Combined tests in 408b270d823e7b2f4da25267120d3ddc8961fa65

Reply:
Thanks, @adamsilverstein!